forge-gui/res/cardsfolder/upcoming/soul_of_windgrace.txt

```diff
@@ -0,0 +1,14 @@
+Name:Soul of Windgrace
+ManaCost:1 B R G
+Types:Legendary Creature Cat Avatar
+PT:5/4
+T:Mode$ ChangesZone | Origin$ Any | OptionalDecider$ You | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigChangeZone | TriggerDescription$ Whenever CARDNAME enters the battlefield, you may put a land card from a graveyard onto the battlefield tapped under your control.
+T:Mode$ Attacks | ValidCard$ Card.Self | OptionalDecider$ You | Execute$ TrigChangeZone | TriggerDescription$ Whenever CARDNAME attacks, you may put a land card from a graveyard onto the battlefield tapped under your control.
```

Comment:
add `Secondary$ True`, then make the TriggerDescription$ here and in line 5 identical to the Oracle